### ISSUE TITLE
Small visual glitches in menus during combat in Chrono Cross

### ISSUE THREAD

**Issue by mckellenjaundice122** [duckstation-unoff-issue-tracker-admin/duckstation-unofficial-issue-tracker#114]
Tested with every output mode available on my PC, DirectX11, DirectX12 and OpenGL. It's easy to reproduce, and i have attached the save state of exactly where this bug can be seen along with a screenshot of what it looks like. The hash of the cd image of my copy of chrono cross is (Chrono Cross Disc 1.bin SHA256: 01A0716ED46B6BCE7717BE1F9220F9BF323CEE52404C9CD7FD4EF747D41A98DA)
![Screenshot 2023-09-22 233716](https://github.com/duckstation-unoff-issue-tracker-admin/duckstation-unofficial-issue-tracker/assets/143659680/ff3c3ef2-70b9-47b9-8e26-de3928a4bf11)

[SLUS-01041_1.zip](https://github.com/duckstation-unoff-issue-tracker-admin/duckstation-unofficial-issue-tracker/files/12704816/SLUS-01041_1.zip)

**Comment by personaone** [comment]
Yup it’s been like this on every possible setting for me as well, interpreter and software renderers affected too

**Comment by sigarettenstank** [comment]
@mckellenjaundice122 Is this fixed now? It doesn't show up for me in the save state. 

![Chrono Cross (USA) (Disc 1) 2024-08-19-11-04-28](https://github.com/user-attachments/assets/9449e6d5-826d-46d7-8a72-ff36118697ed)

**Comment by personaone** [comment]
I think it was fixed a few weeks ago yup

**Comment by mckellenjaundice122** [comment]
The text indeed seems to be fixed, but now for me the shadows under the characters seem to clip and freak out. This is with default settings and i don't believe this to be vanilla behavior. You can see this much clearly in the video than in a screenshot.
[duckstation-qt-x64-ReleaseLTCG_hJ1DwlVh2N.webm](https://github.com/user-attachments/assets/9ada70ac-c872-4444-8029-6d31fec51053)
![image](https://github.com/user-attachments/assets/7d8dcb4a-2b96-4d7f-aa21-1b387218dd33)

Think i figured it out, seems to be related to the CPU execution mode, the glitch in the video doesn't happen if it's set to (interpreter) slowest, but every other mode brings the bug back, so i'm assuming it has to do something with precision, maybe it's worth looking into.
